### DISCUSSION TITLE
[Directives] can multiple directives exist on a field?

### DISCUSSION THREAD

**Discussion by nyex-cell** [zieka/graphql-reshape#46] [Q&A] [answered]
Can multiple directives exist on a field ?

```gql
type Foo {
  bar: String! @directive1 @directive2
}
```

**Comment by zieka** [comment, marked as answer]
Yes, multiple schema directives can exist on a field.  I believe multiple directives can exist anywhere a valid directives location is. There are also many locations so you can refactor lots of repetitive logic into composable directives to make for clean schemas.

More info about directives can be found in the official spec here:
https://spec.graphql.org/June2018/#sec-Validation.Directives
